frontend/src/components/pages/Dashboard/components/DropoffCard.tsx

```diff
@@ -1,131 +1,110 @@
-      onClick={() => history.push(`${Routes.DASHBOARD_PAGE}/${schedule.id}`)}
+      borderLeft={`7px solid ${getFrequencyColor(frequency)}`}
+      boxShadow="2px 2px 12px rgba(0, 0, 0, 0.08)"
+      width={{ base: "default", md: "100%" }}
+      onClick={() => history.push(`${Routes.DASHBOARD_PAGE}/${id}`)}
     >
-      <DeleteScheduleModal
-        isOpen={isOpen}
-        onClose={onClose}
-        onDelete={() => {
-          onDelete();
-          onClose();
-        }}
-      />
-      <Box pl="6" pr="6" pb="6" pt="4">
-        <Box spacing="0" display="flex">
-          <Text
-            mt="0.5rem"
-            mb="16px"
-            textStyle={isDesktop ? "desktopSubtitle" : "mobileBodyBold"}
-            whiteSpace="nowrap"
-            flexGrow={8}
-          >
-            {startDate}
-          </Text>
-          <Box marginLeft="0px">
-            <Menu isLazy>
-              <MenuButton
-                style={{
-                  marginLeft: "6rem",
-                  marginBottom: "15px",
-                  marginRight: "0px",
-                }}
-                as={IconButton}
-                aria-label="options"
-                icon={<EllipsisIcon />}
-                variant="ghost"
-                backgroundColor="transparent"
-                onClick={(e) => e.stopPropagation()}
-              />
-              <MenuList p={0} minW="0" w="94px">
-                <MenuItem
-                  onClick={() =>
-                    history.push(`${Routes.DASHBOARD_PAGE}/${schedule.id}`)
-                  }
-                  textStyle="mobileSmall"
-                >
-                  Edit
-                </MenuItem>
-                <MenuItem
-                  textStyle="mobileSmall"
-                  color="tomato.100"
-                  onClick={(e) => {
-                    e.stopPropagation();
-                    onOpen();
-                  }}
-                >
-                  Cancel
-                </MenuItem>
-              </MenuList>
-            </Menu>
-          </Box>
-        </Box>
-        <HStack>
-          <TimeIcon color="black.100" />
-          <Text textStyle="mobileBodyBold">Time: </Text>
-          <Text textStyle="mobileBody">{startTime}</Text>
-        </HStack>
+      <Stack
+        direction={["column", "row"]}
+        pl="6"
+        pr="6"
+        pb="6"
+        pt="4"
+        display={["default", "flex"]}
+        spacing={["0", "4"]}
+        alignItems="center"
+      >
+        <Text
+          minWidth="150px"
+          textTransform="uppercase"
+          textStyle="mobileSmall"
+          color="hubbard.100"
+          pb={["6px", "0px"]}
+        >
+          {dateHeadingText(startDateLocal)}
+        </Text>
+        <Text
+          textStyle="mobileHeader4"
+          whiteSpace="nowrap"
+          minWidth="250px"
+          pb={["18px", "0px"]}
+        >
+          {`${startTimeLocal}-${endTimeLocal}`}
+        </Text>
+        {volunteerNeeded && (
```

Comment:
I was previously taught that booleans in this format should have the !! ahead of it so: ```!!volunteerNeeded``` to account for possibly undefined behaviour - just a nitpick though because I don't know if we will get to this situation.

Reply:
good note! However, `!!` is usually used to cast a non-boolean variable to a boolean for checking (for instance, a string or a number)  --> in our case, since `volunteerNeeded` is already of type boolean, we don't need to add it. Additionally, this is a **mandatory** type (you can see the scheduling model) so it'll always have a value :)

Reply:
yea just saw that haha woops my bad - thanks for the following up!